src/modules/roc_fec/writer.cpp

```diff
@@ -14,6 +14,14 @@
 namespace roc {
 namespace fec {
 
+namespace {
+
+packet::seqnum_t get_random_seqnum() {
+    return (packet::seqnum_t)core::random(packet::seqnum_t(-1));
+}
+
+} // namespace
+
 Writer::Writer(const Config& config,
```

Comment:
Could you add a test that checks how the Writer fills SNs and SBNs on produced packets?

Reply:
Done